tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_Fluxes.cpp

```diff
@@ -0,0 +1,23 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Fluxes.hpp"
+#include "Evolution/Systems/RadiationTransport/Tags.hpp"  // IWYU pragma: keep
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+
+SPECTRE_TEST_CASE("Unit.RadiationTransport.M1Grey.Fluxes", "[Unit][M1Grey]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{
```

Comment:
Is there an analytic solution that could be tested in addition to the python tests?

Reply:
Not really at the moment - I may derive one when doing transport in a Schwarzschild background and use it as a test for both this and the sources...

Reply:
Okay, sounds good!